src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.ps1

```diff
@@ -413,4 +413,64 @@ function Test-VirtualNetworkGatewayActiveActiveFeatureOperations
         # Cleanup
         Clean-ResourceGroup $rgname
      }
+}
+
+<#
+.SYNOPSIS
+Virtual network gateway BGP route API test
+#>
+function Test-VirtualNetworkGatewayBgpRouteApi
```

Comment:
@lamchester are there any asserts needed to check the output of a given cmdlet in this test? It looks like it relies entirely on the cmdlets not throwing when executed; we want to make sure the output of the cmdlets are items that we expect and aren't just random.